langchain_openai_api_bridge/core/utils/pydantic_async_iterator.py

```diff
@@ -5,4 +5,4 @@
 
 async def ato_dict(async_iter: AsyncIterator[BaseModel]) -> AsyncIterator[dict]:
     async for obj in async_iter:
-        yield obj.dict()
```

Comment:
Since langchain use pydantic V1 by default. `.dict` should be kept until `langchain` use V2 by default.